apps/web/app/(basenames)/name/[username]/page.tsx

```diff
@@ -53,7 +50,9 @@ export default async function Username({ params }: UsernameProfileProps) {
     <ErrorsProvider context="profile">
       <ProfileProviders username={username} address={ensAddress}>
         <main className={usernameProfilePageClasses}>
-          <UsernameProfile />
+          <div className="w-full">
+            <UsernameProfile />
+          </div>
```

Comment:
nit: should this w-full be on the root element of the `UsernameProfile` component?